Album.html

```diff
@@ -3,67 +3,97 @@
+            box-shadow: 0 0 0 0;
         }
     </style>
 </head>
 
 <body>
-    <div class="display">
-        <form name="form1">
-            <input type="button" value="Load album" />
-        </form>
+    <div class="position: sticky-top">
+        <button type="button" class=" btn btn-light border-dark" style="height: 50px; margin:15px; padding:15px;box-shadow: 0 4px 4px 2px grey;border-radius:35px">Load album</button>
+        <p>
+            <button type="button" class=" btn btn-light border-dark" style="height: 50px; margin:15px; padding:15px;box-shadow: 0 4px 4px 2px grey;border-radius:35px"><a href="#" id="loa">  Show more</a></button>
+        </p>
     </div>
+
     <div class="display">
-        <div class="container">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-        </div>
-        <div class="container">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-        </div>
-        <div class="container">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-        </div>
-        <div class="container">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
-            <img src="https://i.pinimg.com/originals/76/6c/f7/766cf770ea8dd3529bd8e0c41d6784be.jpg" width="75px" height="75px">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+        <script>
+            $(function(e) {
+                $("DOMElementLoaded").slice(0, 3).show;
+                $("#loa").click(function(e) {
+                    e.preventDefault();
+                    $("DOMElementLoaded:hidden").slice(0, 3).slideDown();
+
+                })
+            })
+        </script>
+        <div>
+
+            <script>
+                document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
this is not correct implementation of ready function. Pay attention at JQuery ready function realization and change here.

Reply:
Done